pkg/controller/nodeipam/ipam/cloud_cidr_allocator.go

```diff
@@ -57,6 +58,17 @@ import (
 
 const workqueueName = "cloudCIDRAllocator"
 
+// clusterStackType represents the cluster's IP family as per
+// https://kubernetes.io/docs/concepts/cluster-administration/networking/#cluster-network-ipfamilies
+type clusterStackType int32
```

Comment:
if we use a string we'll be able to use it in the errors messages, we don't need to optimize for memory usage here

Reply:
Done